conduct.md

```diff
@@ -212,4 +214,8 @@ Any decision made by the instructor for the given infraction(s) will be final.
 ## Names of Individuals who Agree to Adhere to the Guide
 
 Janyl Jumadinova
-:heavy_exclamation_mark: TODO: Add your name
```

Comment:
The TODO here should not be removed. This is to notify students from other groups to also add their names

Reply:
Since this will likely be the last PR and now that all students had a chance to add their names, you could remove this TODO. Sorry for the confusion